index.html

```diff
@@ -11,11 +11,73 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header-class">
+        <section class="logo">
+           <a href="#"><img src="/img/karma-logo.svg" alt="logo"></a> 
+        </section>
+        <nav class="navbar">
+            <ul>
+                <li><a href="#">Meet Karma</a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+     </header>
+    <main>
+       
+
+        <section class="background-image">
+            <!-- <img class="main-image" src="./img/first-background.jpg" alt="ipad,device,man"> -->
+            <section class="background-text">
+                <h1> Introducing Karma</h1>
+                <h3>Bring WiFi with you, everywhere you go.</h3>
+                <button>Learn More</button>
+            </section>
+            
+           
+        </section>
+        <section class="strapline">
+        <p>Everyone needs a little Karma.</p>
+        </section>
+
+        <section class="features">
+        <figure >
+            <img src="./img/icon-devices.svg" alt="devices">
+            <figcaption>Internet for all devices</figcaption>
+        </figure>
+
```

Comment:
```suggestion
```

Reply:
I don't understand this suggestion. Please explain if you could.